src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -1070,11 +1382,23 @@ public void testStoreRefreshTokenSingleUse(TestContext context) {
         .compose(a -> ts.checkTokenNotRevoked(rt2))
         .onComplete(context.asyncAssertSuccess()) // First check should succeed.
         .compose(a -> ts.checkTokenNotRevoked(rt2))
-        .onComplete(context.asyncAssertFailure(b -> assertThat(b.getMessage(), containsString("revoked"))))
+        .onComplete(context.asyncAssertFailure(b -> {
+          assertThat(((TokenValidationException)b).getHttpResponseCode(), is(401));
+          assertThat(b.getMessage(), containsString("leaked"));
+          assertThat(b.getMessage(), containsString("revoked"));
+        }))
         .compose(a -> ts.checkTokenNotRevoked(rt1))
-        .onComplete(context.asyncAssertFailure(b -> assertThat(b.getMessage(), containsString("revoked"))))
+        .onComplete(context.asyncAssertFailure(b -> {
+          assertThat(((TokenValidationException)b).getHttpResponseCode(), is(401));
+          assertThat(b.getMessage(), containsString("leaked"));
+          assertThat(b.getMessage(), containsString("revoked"));
+        }))
         .compose(a -> ts.checkTokenNotRevoked(rt3))
-        .onComplete(context.asyncAssertFailure(b -> assertThat(b.getMessage(), containsString("revoked"))));
+        .onComplete(context.asyncAssertFailure(b -> {
+          assertThat(((TokenValidationException)b).getHttpResponseCode(), is(401));
+          assertThat(b.getMessage(), containsString("leaked"));
+          assertThat(b.getMessage(), containsString("revoked"));
```

Comment:
Can you extract this duplicated code into a method like `assertLeakedOrRevoked(Throwable t)`?